libs/pages/elewa/news/src/lib/components/elewa-news-section/elewa-news-section.component.ts

```diff
@@ -0,0 +1,47 @@
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTtMBs2IlGh-6A0MUjuvzlEcgCjHKZwEIe1Jf-a4888atbcHQvsUfoTwxPjIQgw4whSOhE&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'venture-labs'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR-N12wOMy-iPxJQOg_D_FHK5Qsu1NoHVNBHA&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'press'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR-N12wOMy-iPxJQOg_D_FHK5Qsu1NoHVNBHA&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'elewa'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTA0a7e50cvXyNQ1KcxZ2zpyX5DGlGVK13i6w&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'italanta'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTA0a7e50cvXyNQ1KcxZ2zpyX5DGlGVK13i6w&usqp=CAU', timestamp: '11:30', title: `This is Awesome`, body: '', topic: 'elewa'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR-N12wOMy-iPxJQOg_D_FHK5Qsu1NoHVNBHA&usqp=CAU', timestamp: '11:30', title: `This is Awesome`, body: '', topic: 'italanta'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTA0a7e50cvXyNQ1KcxZ2zpyX5DGlGVK13i6w&usqp=CAU', timestamp: '12:13', title: `This is Awesome`, body: '', topic: 'venture-labs'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTtMBs2IlGh-6A0MUjuvzlEcgCjHKZwEIe1Jf-a4888atbcHQvsUfoTwxPjIQgw4whSOhE&usqp=CAU', timestamp: '10:56', title: `This is Awesome`, body: '', topic: 'press'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR-N12wOMy-iPxJQOg_D_FHK5Qsu1NoHVNBHA&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'elewa'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTA0a7e50cvXyNQ1KcxZ2zpyX5DGlGVK13i6w&usqp=CAU', timestamp: '08:43', title: `And the light`, body: '', topic: 'italanta'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTtMBs2IlGh-6A0MUjuvzlEcgCjHKZwEIe1Jf-a4888atbcHQvsUfoTwxPjIQgw4whSOhE&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'venture-labs'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR-N12wOMy-iPxJQOg_D_FHK5Qsu1NoHVNBHA&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'press'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR-N12wOMy-iPxJQOg_D_FHK5Qsu1NoHVNBHA&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'elewa'},
+      {image: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTtMBs2IlGh-6A0MUjuvzlEcgCjHKZwEIe1Jf-a4888atbcHQvsUfoTwxPjIQgw4whSOhE&usqp=CAU', timestamp: '08:43', title: `This is Awesome`, body: '', topic: 'italanta'}
+    ]
+
+  @Input() activeTopic = 'all-news'
+
+  getNews() :{image: string, timestamp: string, title: string, body: string}[] {
```

Comment:
The return type of this function is Article[]

Reply:
The design of your tabs is not the same as the design provided. Please check design